doc/sphinx-guides/source/_static/util/default.config

```diff
@@ -3,7 +3,7 @@ GLASSFISH_DIRECTORY	/usr/local/glassfish4
 ADMIN_EMAIL	 
 MAIL_SERVER	mail.hmdc.harvard.edu
 POSTGRES_ADMIN_PASSWORD	secret
-POSTGRES_SERVER	127.0.0.1
+POSTGRES_SERVER	dataverse-postgresql-0.dataverse-postgresql-service
```

Comment:
This changes the default behavior (and default documentation example) to openshift - this change might be better to make in `conf/docker/dataverse-glassfish/default.config`.

Reply:
I understand. I believe I have fixed this now.

Reply:
@patrickdillon Thanks! Looks fixed to me.